### PR TITLE
feat: add int64 param

### PR DESCRIPTION
**Pull request by wey-gu**
close: https://github.com/vesoft-inc/nebula-go/issues/288

for int that overflows int64, we leave it to server side's evaluation, instead of doing sanity check in client side now.

Also, the GitHub action runner images are changed recently, docker-compose v1 were dropped thus we have to use `docker compose`(v2), where unnamed container's naming conventions rule changed, thus hardcoding them with names honer previous naming rules were made to ensure tests working fine.

### CODE REVIEW

File: session.go [hunk 1]
```diff
@@ -374,58 +374,61 @@ func map2Nmap(m map[string]interface{}) (*nebula.NMap, error) {
 // construct go-type to nebula.Value
 func value2Nvalue(any interface{}) (value *nebula.Value, err error) {
```
Comment (HarrisChu) [inline session.go:375]: prefer do not use `any`, because it's a keyword now...
it's involved a long time ago, maybe we could change it in this pr?

**Review by HarrisChu — DISMISSED** [review]

File: session.go (hunk 1 above)
Comment (wey-gu) [reply session.go:375]: make sense will use a decent name later, how about param?

Comment (wey-gu) [reply session.go:375]: On a flight being taking off, will do it after landing.

Comment (wey-gu) [reply session.go]: @HarrisChu it's done :)

**Review by HarrisChu — APPROVED** [review]

**Review by wenhaocs — APPROVED** [review]
LGTM

**Review by Nicole00 — APPROVED** [review]

**Pull request merged**